test/addons/chrome/install.rdf

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+    <Description about="urn:mozilla:install-manifest">
+          <em:id>test-chrome@jetpack</em:id>
+          <em:type>2</em:type>
+          <em:bootstrap>true</em:bootstrap>
+          <em:unpack>false</em:unpack>
+          <em:version>0.0.1</em:version>
+          <em:name>Untitled</em:name>
+          <em:description/>
+          <em:creator/>
+          <em:iconURL>icon.png</em:iconURL>
+          <em:icon64URL>icon64.png</em:icon64URL>
+
+          <em:targetApplication>
+            <Description>
+              <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+              <em:minVersion>33.0a1</em:minVersion>
+              <em:maxVersion>38.0</em:maxVersion>
+</Description>
+</em:targetApplication>
+
+
+    </Description>
+
+</RDF>
```

Comment:
Uhh no, in fact I have no idea where it or bootstrap.js came from as I didn't write them.